feature-libs/cart/saved-cart/components/list/saved-cart-list.component.html

```diff
@@ -1,151 +1,159 @@
 <ng-container *ngIf="savedCarts$ | async as savedCarts">
-  <div class="cx-saved-cart-list-header">
-    <h3>
-      {{
-        'savedCartList.savedCarts' | cxTranslate: { count: savedCarts.length }
-      }}
-    </h3>
+  <div *ngIf="isLoading$ | async; else savedCartListContainer">
+    <div class="cx-spinner">
+      <cx-spinner></cx-spinner>
+    </div>
   </div>
```

Comment:
1. For sake of consistency with all other templates, let's reverse IF and make loading spinner as else condition
2. For sake of consistency with all other templates, let's move spinner at the end of file